index.bs

```diff
@@ -2819,6 +2819,13 @@ Maturity Levels on the Recommendation Track</h4>
 					See also “W3C Royalty-Free (RF) Licensing Requirements”
 					in the W3C Patent Policy [[!PATENT-POLICY]].
 			</dl>
+		<dt>
+			<dfn id="discontinuedREC" export >Discontinued Draft Recommendation</dfn> (<abbr>REC</abbr>)
```

Comment:
I'm very supportive of the direction of change; the name "Discontinued Draft Recommendation" feels suboptimal: too easy to confuse with Obsolete or Rescinded Recommendation when they are in practice very different. “Discontinued Draft" is both shorter and less confusion prone. (I'm also guessing REC would not be a good abbreviation for that status no matter what >:)